### PR TITLE
 Modified export to be compatible with LCSC BOM

### PR DESCRIPTION
**Pull request by darkomen**
I have modified the CSV to be compatible with LCSC BOM tool (www.lcsc.com) . In this manner, we can export our CSV and automatically the web will generate the order.

### CODE REVIEW

File: bomsaway.py [hunk 1]
```diff
@@ -546,21 +546,21 @@ def on_export(self, event):
         with open(base+ext, 'w') as csvfile:
             wrt = csv.writer(csvfile)
 
-            wrt.writerow(['Refs', 'Value', 'Footprint',
-                          'QTY', 'MFR', 'MPN', 'SPR', 'SPN'])
+            wrt.writerow(['Quantity','Value', 'Refs', 'Footprint',
+                           'Manufacturer','Manufacture Part Number',
+                           'LCSC Part Number'])
```
Comment (Jeff-Ciesielski) [inline bomsaway.py:551]: Sorry, but I'd rather not hardcode the name of a supplier into the code (SPN == Supplier Part Number).

Comment (Jeff-Ciesielski) [reply bomsaway.py:551]: If LCSC can handle a more generic field name, the rest of this looks fine.

Comment (Jeff-Ciesielski) [reply bomsaway.py:551]: @darkomen : can you give https://github.com/Jeff-Ciesielski/Boms-Away/commit/8c9f02d8d30a336ca8d494bf3a8f24f70fb5e942 a try and see if the LCSC.com exporter works?

Comment (darkomen) [reply bomsaway.py:551]: Hi Jeff! yes you commit works correctly! You're right, it's better in this way. Certainly I doesn't try with this format :)

**Comment by Jeff-Ciesielski** [comment]
Ok, closing this since the new import plugin works.  Thank you for the PR though :) It spurred things in the right direction for sure :)

**Pull request closed without merging**